felles/src/main/kotlin/no/nav/familie/kontrakter/felles/oppgave/Oppgave.kt

```diff
@@ -7,38 +7,53 @@ import javax.validation.constraints.Pattern
-                   val saksreferanse: String? = null,
-                   val bnr: String? = null,
-                   val samhandlernr: String? = null,
-                   @field:Pattern(regexp = "[0-9]{13}")
-                   val aktoerId: String? = null,
-                   val orgnr: String? = null,
-                   val tilordnetRessurs: String? = null,
-                   val beskrivelse: String? = null,
-                   val temagruppe: String? = null,
-                   val tema: Tema? = null,
-                   val behandlingstema: String? = null,
-                   val oppgavetype: String? = null,
-                   val behandlingstype: String? = null,
-                   val versjon: Int? = null,
-                   val mappeId: Long? = null,
-                   val fristFerdigstillelse: String? = null,
-                   val aktivDato: String? = null,
-                   val opprettetTidspunkt: String? = null,
-                   val opprettetAv: String? = null,
-                   val endretAv: String? = null,
-                   val ferdigstiltTidspunkt: String? = null,
-                   val endretTidspunkt: String? = null,
-                   val prioritet: OppgavePrioritet? = null,
-                   val status: StatusEnum? = null,
-                   private var metadata: MutableMap<String, String>? = null)
+        val identer: List<OppgaveIdentV2>? = null,
+        val tildeltEnhetsnr: String? = null,
+        val endretAvEnhetsnr: String? = null,
+        val opprettetAvEnhetsnr: String? = null,
+        val journalpostId: String? = null,
+        val journalpostkilde: String? = null,
+        val behandlesAvApplikasjon: String? = null,
+        val saksreferanse: String? = null,
+        val bnr: String? = null,
+        val samhandlernr: String? = null,
+        @field:Pattern(regexp = "[0-9]{13}")
+        val aktoerId: String? = null,
+        val orgnr: String? = null,
+        val tilordnetRessurs: String? = null,
+        val beskrivelse: String? = null,
+        val temagruppe: String? = null,
+        val tema: Tema? = null,
+        val behandlingstema: String? = null,
+        val oppgavetype: String? = null,
+        val behandlingstype: String? = null,
+        val versjon: Int? = null,
+        val mappeId: Long? = null,
+        val fristFerdigstillelse: String? = null,
+        val aktivDato: String? = null,
+        val opprettetTidspunkt: String? = null,
+        val opprettetAv: String? = null,
+        val endretAv: String? = null,
+        val ferdigstiltTidspunkt: String? = null,
+        val endretTidspunkt: String? = null,
+        val prioritet: OppgavePrioritet? = null,
+        val status: StatusEnum? = null,
+        private var metadata: MutableMap<String, String>? = null,
+) {
+
+    fun leggTilMetadata(key: String, verdi: String) {
+        if (metadata == null) {
+            metadata = mutableMapOf()
+        }
+        metadata!![key] = verdi
```

Comment:
```suggestion
        metadata[key] = verdi
```

Reply:
Kotlin liker ikke den - mutable map kan ha blitt endret

Reply:
-> Smart cast to 'MutableMap<String, String>' is impossible, because 'metadata' is a mutable property that could have been changed by this time

Reply:
Ah ja. Hvorfor skal man kunne endre metadata i en oppgave?

Reply:
Dette er en PoC for å sjekke om vi kan bruke metadata for å kommunisere med oppgavebenk. Er dette en oppgave vi kan starte med behandling i ny løsning (i dette tilfellet en blankettbehandling).

Vi ønsker å legge inn en enkel "kanBlankettBehandles=true" på oppgaver som har en fagsakId i infotrygd.

Dette gjør at vi kan identifisere disse oppgavene i oppgavebenken vår.